### ISSUE TITLE
macOS: Mono still 32bit (Warning on launch)

### ISSUE THREAD

**Issue by mazimmer** [duplicati/duplicati#3854]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.4.23
- **Operating system**: macOS 10.14
- **Backend**: n/a

## Description
macOS warns that the mono binary is still 32bit, which won't run from 10.15

- **Actual result**:
32bit warning
- **Expected result**:
64bit binary, no warning

**Comment by aahung** [comment]
I don't seem to see any warning about 32-bit on my Mac. 
And mono is 64-bit here:

<img width="643" alt="image" src="https://user-images.githubusercontent.com/3804518/63120741-c0402500-bf57-11e9-8064-2309b6b0ec3e.png">

**Comment by mazimmer** [comment]
Mine is 32bit. 
<img width="812" alt="Screenshot 2019-08-15 at 22 30 15" src="https://user-images.githubusercontent.com/4983700/63124966-cad8d980-bfac-11e9-97ec-111a8df95cfb.png">

 > marco@Hermes /A/D/C/MacOS> /Library/Frameworks/Mono.framework/Versions/4.2.3/bin/mono-sgen --version
> Mono JIT compiler version 4.2.3 (explicit/832de4b Thu Mar  3 19:24:57 EST 2016)
> Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
> 	TLS:           normal
> 	SIGSEGV:       altstack
> 	Notification:  kqueue
> 	Architecture:  x86
> 	Disabled:      none
> 	Misc:          softdebug
> 	LLVM:          yes(3.6.0svn-mono-(detached/a173357)
> 	GC:            sgen

But then that's the problem I guess: I had assumed Duplicati installs the Mono Runtime, looks like I've done that separately once and forgotten about it. 

So, sorry about this. 

Maybe Duplicati should bundle it, though?

**Comment by Pectojin** [comment]
The tricky part about bundling mono is that we can't update mono. 

This is the case since many people use the built in duplicati updater, which runs within mono,

**Comment by aahung** [comment]
The update experience of mono is not seamless at all, adding even more friction.

**Comment by ts678** [comment]
> But then that's the problem I guess: I had assumed Duplicati installs the Mono Runtime, looks like I've done that separately once and forgotten about it.

`mono` is still a do-it-yourself install/upgrade for macOS. Install got harder now that macOS 12.3 removed Python 2.7 we used. [Installation](https://duplicati.readthedocs.io/en/latest/02-installation/) section of the manual helps a little with traditional mono installation, but hasn't yet been updated for Python 3 etc.

Things are definitely not ideal, and additional issues can be opened, but some friction should will go away if Duplicati moves to current `.NET` technologies. See [Introducing .NET 5](https://devblogs.microsoft.com/dotnet/introducing-net-5/) for where things are going in the unification plan that leaves `mono` behind.

Question is what to do with this issue? I've set it up to automatically close unless someone wants to post more to keep it open.
